<ui version="4.0" >
 <class>DatabaseSettingsDlg</class>
 <widget class="QDialog" name="DatabaseSettingsDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>213</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>440</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Database Settings</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox1" >
     <property name="title" >
      <string>Encryption</string>
     </property>
     <layout class="QGridLayout" >
      <property name="horizontalSpacing" >
       <number>15</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="textLabel2" >
        <property name="text" >
         <string>Algorithm:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="ComboAlgo" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="textLabel3" >
        <property name="text" >
         <string>Encryption Rounds:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLineEdit" name="EditRounds" />
        </item>
        <item>
         <widget class="QPushButton" name="ButtonBench" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip" >
           <string>Calculate rounds for a 1-second delay on this computer</string>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ButtonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>ComboAlgo</tabstop>
  <tabstop>EditRounds</tabstop>
  <tabstop>ButtonBench</tabstop>
  <tabstop>ButtonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
